test/test_bagitutils.py

```diff
@@ -112,8 +111,11 @@ def test_fc_mock(self):
         self.assertEqual(max_files_in_sample, 3)
 
         samples = bag.samples(max_files_in_sample, protocols)
+        self.assertEqual(len(samples[0].keys()), 22)
         self.assertEqual(len(samples), 6)
 
+        bag.write_csv_files('/home/michael/dev/junk')
+
```

Comment:
Hard-coded to your home directory, plus it looks like this is a debugging line you accidentally left in?

Reply:
all of the commented lines / this hard-coded line were unintentionally left in, I took all of them out